tests/start_network.py

```diff
@@ -91,6 +107,11 @@ def add(parser):
             action="store_true",
             default=False,
         )
+        parser.add_argument(
+            "--client-info-file",
+            help="Path to output file where client information will be dumped to (useful for scripting)",
+            default="client_info.txt",
```

Comment:
Minor: I think client-info is a bad name, this is really `node_address` or something? Or maybe `node_info` or `service_info` (given "foo_info", I think an implicit "info about Foo" is more obvious than a "info _for_ Foo").

Anyway this can be default=None, check it looks like a path if provided, and we only write the file if it was specified?

Reply:
Good point. I have updated the name of the file and changed the default behaviour.